### PR TITLE
Cleanup transaction schema

### PR DESCRIPTION
**Pull request by Ifropc**
<details>
  <summary>PR Checklist</summary>

### PR Structure

* [ ] This PR has reasonably narrow scope (if not, break it down into smaller PRs).
* [ ] This PR avoids mixing refactoring changes with feature changes (split into two PRs
  otherwise).
* [ ] This PR's title starts with name of package that is most changed in the PR, ex.
  `paymentservice.stellar`, or `all` or `doc` if the changes are broad or impact many
  packages.

### Thoroughness

* [ ] This PR adds tests for the most critical parts of the new functionality or fixes.
</details>

### What

1. Remove kycVerified 
2. Update transferReceivedAt only for SEP-31
3. Remove custodialTransactionId

### Why

1. Not used 
2. Was only used by SEP-31
3. Not used

### Known limitations

N/A

### CODE REVIEW

File: platform/src/main/java/org/stellar/anchor/platform/service/TransactionService.java [hunk 1]
```diff
@@ -230,6 +228,8 @@ void updateSepTransaction(PlatformTransactionData patch, JdbcSepTransaction txn)
         }
         break;
       case "31":
+        txnUpdated = updateField(patch, txn, "transferReceivedAt", txnUpdated);
+
```
Comment (JakeUrban) [inline platform/src/main/java/org/stellar/anchor/platform/service/TransactionService.java:232]: Do we not want to update this field in SEP-24 withdrawals?

File: api-schema/src/main/java/org/stellar/anchor/api/platform/PlatformTransactionData.java [hunk 2]
```diff
@@ -66,10 +63,6 @@ public class PlatformTransactionData {
   @SerializedName("external_transaction_id")
   String externalTransactionId;
 
-  // TODO: review the use case of the custodialTransactionId
-  @SerializedName("custodial_transaction_id")
-  String custodialTransactionId;
```
Comment (JakeUrban) [inline api-schema/src/main/java/org/stellar/anchor/api/platform/PlatformTransactionData.java:71]: I don't see this field being removed in a migration file. Did it not exist in the first place?

**Review by JakeUrban — COMMENTED** [review]
Was the V6 migration file added for the 2.0 release or did it exist for 1.x releases? If it existed in 1.x releases we should make a new migration file for removing the column instead of deleting the add statement.

File: platform/src/main/java/org/stellar/anchor/platform/service/TransactionService.java (hunk 1 above)
Comment (Ifropc) [reply platform/src/main/java/org/stellar/anchor/platform/service/TransactionService.java:232]: It's not returned in GET response of SEP-24, so I don't think it makes sense to update it at all

File: api-schema/src/main/java/org/stellar/anchor/api/platform/PlatformTransactionData.java (hunk 2 above)
Comment (Ifropc) [reply api-schema/src/main/java/org/stellar/anchor/api/platform/PlatformTransactionData.java:71]: It's never been a part of initial SQL script

File: platform/src/main/java/org/stellar/anchor/platform/service/TransactionService.java (hunk 1 above)
Comment (JakeUrban) [reply platform/src/main/java/org/stellar/anchor/platform/service/TransactionService.java:232]: It should be in the Platform API's transaction response though

**Comment by Ifropc** [comment]
> Was the V6 migration file added for the 2.0 release or did it exist for 1.x releases? If it existed in 1.x releases we should make a new migration file for removing the column instead of deleting the add statement.

It's for v2

**Review by JakeUrban — APPROVED** [review]

**Pull request merged**